cli/Sources/TuistSupport/Constants.swift

```diff
@@ -62,8 +62,6 @@ public enum Constants {
         /// `tuistBinaryPath` is used for specifying the exact tuist binary in tuist tasks.
         public static let tuistBinaryPath = "TUIST_CONFIG_BINARY_PATH"
         public static let token = "TUIST_CONFIG_TOKEN"
-        @available(*, deprecated, message: "Use `token` instead")
-        public static let deprecatedToken = "TUIST_CONFIG_CLOUD_TOKEN"
```

Comment:
should we communicate this? I'm not sure how many organizations still use this one – hopefully none – but can we be sure that's the case?

Reply:
I see, I missed that, never mind then 👍